### PR TITLE
Given example format for env.bosh.group[s] in CPI API doc

### PR DESCRIPTION
**Pull request by cunnie**
Signed-off-by: Brian Cunnie bcunnie@pivotal.io

[#127510881](https://www.pivotaltracker.com/story/show/127510881)

### CODE REVIEW

File: cpi-api-v1.html.md.erb [hunk 1]
```diff
@@ -94,11 +94,12 @@ Make sure to properly delete created resources if VM cannot be successfully crea
 - **cloud_properties** [Hash]: Cloud properties hash specified in the deployment manifest under VM's resource pool.
 - **networks** [Hash]: Networks hash that specifies which VM networks must be configured.
 - **disk_cids** [Array of strings] Array of disk cloud IDs for each disk that created VM will most _likely_ be attached; they could be used to optimize VM placement so that disks are located nearby.
-- **environment** [Hash]: Resource pool's env hash specified in deployment manifest.
+- **environment** [Hash]: Resource pool's env hash specified in deployment manifest including
+initial properties added by the BOSH director as shown below.
```
Comment (dpb587-pivotal) [inline cpi-api-v1.html.md.erb]: [Apparently](https://github.com/cloudfoundry/docs-bosh/pull/223#issuecomment-253623469) we're not supposed to wrap lines in these docs.

Comment (cppforlife) [inline cpi-api-v1.html.md.erb]: yup. dont wrap.

File: cpi-api-v1.html.md.erb [hunk 2]
```diff
@@ -115,15 +116,27 @@ Make sure to properly delete created resources if VM cannot be successfully crea
...
 			"cloud_properties": {}
 		}
 	},
 	[ "vol-3475945" ],
-	{}
+	{
+		"bosh": {
+			"group": "bosh_director_name-deployment_name-instance_group_name",
+			"groups": [
+				"bosh_director_name",
```
Comment (cppforlife) [inline cpi-api-v1.html.md.erb]: let's not add actual names since they should be opaque. otherwise it's a bit misleading in terms of promising some format.

**Comment by ljfranklin** [comment]
We have "unwrapped" the long line.

We have made the group names generic, lest people are tempted to assume the Security Group names are formats that are set in stone.

**Pull request merged**